### PR TITLE
Add FAQ to the mysql deprecation

### PR DESCRIPTION
**Pull request by isacikgoz**
#### Summary
To kick this effort off, I'm creating the PR so that anyone can also contribute from here.

To be doc'd:
- RDS Aurora MySQL > RDS PostgreSQL fully supported. No tool limitations.
- Migrating a Bitnami install (defaults to MySQL) requires Mattermost binary transition from TE to E0 & custom DB migration effort. 
- No checks for whether ``–migrations-dir`` is provided. ``–mattermost-version`` is required if not provided. 
- Blocking error could not check schema owner: mmuser isn’t the owner of the public schema. Manual workaround required.
- workaround steps for cases where [search_path](https://docs.mattermost.com/deploy/manual-postgres-migration.html#restore-the-search-path) is set to null. [context](https://hub.mattermost.com/private-core/pl/6s5q9ex6jp8zpkk65ue7cyiqyo)

### CODE REVIEW

File: source/deploy/postgres-migration.rst [hunk 1]
```diff
@@ -15,4 +15,27 @@ From Mattermost v8.0, PostgreSQL is our database of choice for Mattermost to enh
    Manually migrate to PostgreSQL </deploy/manual-postgres-migration>
 
 * :doc:`Automated migration from MySQL to PostgreSQL </deploy/postgres-migration-assist-tool>` - A comprehensive set of guidelines and a ``migration-assist`` tool to streamline the migration process, alleviate potential challenges, and faciliate a smooth transition.
-* :doc:`Manually migrate from MySQL to PostgreSQL </deploy/manual-postgres-migration>` - A good option if your organization has database administrators to own the migration process, or if you want to learn what the ``migration-assist`` tool automates for you.
\ No newline at end of file
+* :doc:`Manually migrate from MySQL to PostgreSQL </deploy/manual-postgres-migration>` - A good option if your organization has database administrators to own the migration process, or if you want to learn what the ``migration-assist`` tool automates for you.
+
+FAQ
+===
```
Comment (cwarnermm) [inline source/deploy/postgres-migration.rst:21]:
```suggestion
----
```

File: source/deploy/postgres-migration.rst [hunk 2]
```diff
@@ -15,4 +15,27 @@ From Mattermost v8.0, PostgreSQL is our database of choice for Mattermost to enh
    Manually migrate to PostgreSQL </deploy/manual-postgres-migration>
 
 * :doc:`Automated migration from MySQL to PostgreSQL </deploy/postgres-migration-assist-tool>` - A comprehensive set of guidelines and a ``migration-assist`` tool to streamline the migration process, alleviate potential challenges, and faciliate a smooth transition.
-* :doc:`Manually migrate from MySQL to PostgreSQL </deploy/manual-postgres-migration>` - A good option if your organization has database administrators to own the migration process, or if you want to learn what the ``migration-assist`` tool automates for you.
\ No newline at end of file
+* :doc:`Manually migrate from MySQL to PostgreSQL </deploy/manual-postgres-migration>` - A good option if your organization has database administrators to own the migration process, or if you want to learn what the ``migration-assist`` tool automates for you.
+
+FAQ
```
Comment (cwarnermm) [inline source/deploy/postgres-migration.rst:20]:
```suggestion
Frequently asked questions
```

File: source/deploy/postgres-migration.rst [hunk 3]
```diff
@@ -15,4 +15,27 @@ From Mattermost v8.0, PostgreSQL is our database of choice for Mattermost to enh
    Manually migrate to PostgreSQL </deploy/manual-postgres-migration>
 
 * :doc:`Automated migration from MySQL to PostgreSQL </deploy/postgres-migration-assist-tool>` - A comprehensive set of guidelines and a ``migration-assist`` tool to streamline the migration process, alleviate potential challenges, and faciliate a smooth transition.
-* :doc:`Manually migrate from MySQL to PostgreSQL </deploy/manual-postgres-migration>` - A good option if your organization has database administrators to own the migration process, or if you want to learn what the ``migration-assist`` tool automates for you.
\ No newline at end of file
+* :doc:`Manually migrate from MySQL to PostgreSQL </deploy/manual-postgres-migration>` - A good option if your organization has database administrators to own the migration process, or if you want to learn what the ``migration-assist`` tool automates for you.
+
+Frequently asked questions
+----
```
Comment (cwarnermm) [inline source/deploy/postgres-migration.rst:21]:
```suggestion
------------------------------
```

**Comment by isacikgoz** [comment]
@cwarnermm gentle ping for the editorial review :) Also please LMK if this is okay to merge @coltoneshaw @nab-77. I have other PRs incoming :)

**Comment by cwarnermm** [comment]
@isacikgoz - I haven't yet contributed the updates noted in the PR summary above. Wondering if your updates included those or if they're still outstanding? I'm open to moving forward with this PR and adding any outstanding details in a future revision.

File: source/deploy/postgres-migration.rst [hunk 4]
```diff
@@ -17,4 +17,73 @@ From Mattermost v8.0, PostgreSQL is our database of choice for Mattermost to enh
...
+4. Do we/will we bundle pgloader or is that a separate install?
+
+  We do not bundle pgloader with the Mattermost server. You will need to install pgloader separately. For more information, see the :doc:`install pgloader </deploy/manual-postgres-migration:install pgloader>` documentation.
+
+5. Are there any other migrations available for plugins, or just Boards, Playbooks and Calls?
+  
+  We are working on providing migrations for other plugins as well such as NPS-plugin. Please stay tuned for updates.
+
+Troubleshooting
+===============
```
Comment (cwarnermm) [inline source/deploy/postgres-migration.rst:44]: A double-underline is reserved for the top page heading. Subsequent headings are denoted as follows:

- H2: ``------``
- H3: ``~~~~~~``
- H4: ``^^^^^^``
- H5: ``::::::``

**Review by cwarnermm — CHANGES_REQUESTED** [review]
Please update the RST syntax specific to section titles as noted inline.

**Comment by isacikgoz** [comment]
Oh @cwarnermm I haven't seen your additions 😅 

> Migrating a Bitnami install (defaults to MySQL) requires Mattermost binary transition from TE to E0 & custom DB migration effort.

I'll take a look into that, but we can add that in the future.

> No checks for whether –migrations-dir is provided. –mattermost-version is required if not provided.

I'll add this to the next doc update. It's very specific to automated process.

An the rest is done in this PR.

**Review by cwarnermm — APPROVED** [review]
Thanks, @isacikgoz!

**Pull request merged**